recipes/callingcardstools/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "callingcardstools" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 74a20d1eb22f7bdf3373e70e57737caefe4ea63f115e975dba6c04be814288fd
+
+build:
+  noarch: generic
+  number: 0
+  entry_points:
+    - callingcardstools = callingcardstools:__main__.main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - biopython
+    - edlib
+    - numpy
+    - pandas
+    - pip
+    - pysam
+    - poetry
+    - python
```

Comment:
For compatibility, is it possible to specify a version of python here? E.g. `python <3` or `python >=3.6`

Reply:
Yes -- I originally had this specified, and then looked at the pysam meta.yml as an example. There are not specified versions there.

My pyproject.toml has specified software versions. I erred on the side of removing them, under the assumption that the pyproject.toml would act to resolve any issues. However, I wasn't sure.

I added the python specification back, and I'm happy to add specs for the rest of the packages.